### ISSUE TITLE
Need example for working with encryption

### ISSUE THREAD

**Issue by manojhl** [rse/node-prince#4]
I'm able to generate pdf files without encryption, but when I give the encrypt option its not creating a encrypted pdf.

```js
Prince({"encrypt": true}, "key-bits": 128, "user-password": "12345")
    .inputs("sample.html")
    .output("tax/test-"+count+".pdf")
    .execute()
    .then(function () {
        console.log("OK: done");
    }, function (error) {
        console.log("ERROR: ", util.inspect(error));
    })
```

Guess need some examples and documentation on how to use options

**Comment by rse** [comment]
There are no second and third paramater to the Prince constructor. You have to use the "option" method to pass the prince(1) CLI options:

```js
Prince()
    .option("encrypt", true)
    .option("key-bits", 128)
    .option("user-password", "12345")
    .inputs("sample.html")
    .output("tax/test-"+count+".pdf")
    .execute()
    .then(function () {
        console.log("OK: done");
    }, function (error) {
        console.log("ERROR: ", util.inspect(error));
    })
```

**Comment by manojhl** [comment]
oh, my bad. But wouldn't it be nice if we can provide multiple
parameters(options) via Prince() constructor. Anyway thanks for pointing it
out.

On Wed, Aug 26, 2015 at 11:29 PM, Ralf S. Engelschall <
notifications@github.com> wrote:

> Closed #4 <https://github.com/rse/node-prince/issues/4>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rse/node-prince/issues/4#event-393165229>.
>



-- 
Manoj H L
Freelancer
Web Developer @ Greenpeace India Society
